src/back/routes/update.ts

```diff
@@ -31,7 +31,7 @@ import { VestingService } from '../../services/VestingService'
 import { ErrorCategory } from '../../utils/errorCategories'
 import type { Project } from '../models/Project'
 import { UpdateService } from '../services/update'
-import { validateCanEditProject } from '../utils/validations'
+import { validateIsAuthorOrCoauthor } from '../utils/validations'
```

Comment:
You are using the old fn :)

Reply:
I had to divide the validation into 2 parts!